submodules/configure_network_services/templates/services_init.sh.tftpl

```diff
@@ -154,6 +154,12 @@ register_os() {
         done
       fi
 
+     ##### check if the subscription log file /var/log/powervs-fls.log exists
+      if [ ! -e "$SUBSCRIPTION_LOG" ]; then
```

Comment:
in print change it to : Subscription log $SUBSCRIPTION_LOG ,
 and this block should come just before while loop at the bottom inside the ppc64le

Reply:
You mean inside of this block:

 ```
##### check if the system is a ppc64le processor
      if [[ "$ARCH" == "ppc64le" ]]; then

        if [ ! -e "$README" ]; then
          print_out "PER enabled DC. Registration was done automatically."
        else
          #will run the register command
          
          ##### check if the subscription log file /var/log/powervs-fls.log exists
        if [ ! -e "$SUBSCRIPTION_LOG" ]; then
          print_out "Subscription log $SUBSCRIPTION_LOG is missing.Cannot perform OS Registration.OS Registration failed."
          exit 1
        fi

          suseconnect_register
        fi
```

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/blob/bc0bea3d06616c1ea3082a5382f812adfc47bc66/submodules/configure_network_services/templates/services_init.sh.tftpl#L235
add sleep 60 and then insert the block and then while loop follows.

Testing is required .
Deploy an instance. Then login to the OS. Unregister it.
Delete "/usr/share/powervs-fls/powervs-fls-readme.md" and  "/var/log/powervs-fls.log" files. 
Run OS registration the file is in terraform_scripts dir :  services_init register_os 

It should out print:  "Subscription log $SUBSCRIPTION_LOG is missing.Cannot perform OS Registration.OS Registration failed."

Please verify